src/mount.c

```diff
@@ -243,13 +245,46 @@ cc_handle_mounts(struct cc_oci_config *config, GSList *mounts, gboolean volume)
 			continue;
 		}
 
-		if ((! cc_pod_is_vm(config) || cc_pod_is_pod_sandbox(config)) && volume) {
+		if (! volume) {
+			/* bind mount container rootfs. We do this for regular
+			 * container as well for a pod sandbox.
+			 */
```

Comment:
Will change that to include cc_pod_is_vm() check. Although this bind-mounting change with the added check doesnt seem to play well with crio. Still taking a look at this.